packages/roosterjs-content-model-plugins/lib/edit/keyCode.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Key code enumeration
+ */
+export const enum KeyCode {
```

Comment:
Just put it back in EditPlugin.ts and no need to export it.
And use a regular const instead of const enum since const enum has a lot of problem.